app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/ConfigurationPage.vue

```diff
@@ -300,9 +303,24 @@ export default {
     <div class="mx-8">
       <SettingsSection
         :title="$t('INBOX_MGMT.SETTINGS_POPUP.FORWARD_EMAIL_TITLE')"
-        :sub-title="$t('INBOX_MGMT.SETTINGS_POPUP.FORWARD_EMAIL_SUB_TEXT')"
+        :sub-title="
+          hasInboundEmailDomain
+            ? $t('INBOX_MGMT.SETTINGS_POPUP.FORWARD_EMAIL_SUB_TEXT')
+            : ''
+        "
       >
-        <woot-code :script="inbox.forward_to_email" />
+        <woot-code
+          v-if="hasInboundEmailDomain"
```

Comment:
Can't we just add this to inbox.json.jbuilder? 
```rb
json.forwarding_enabled GlobalConfigService.load('MAILER_INBOUND_EMAIL_DOMAIN', '').present?
```

We can then use inbox.forwarding_enabled check